src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Establish network connections early to improve perceived page speed
+subhead: |
+    Learn about rel=preconnect and rel=dns-prefetch resource hints and how to use them.
+date: 2019-07-30
+hero: adams-creation.jpg
+alt: Adam's Creation by Michelangelo on Sistine Chapel ceiling 
+authors:
+  - mihajlija
+description: |
+    Learn about rel=preconnect and rel=dns-prefetch resource hints and how to use them.
+tags:
+  - post
+  - performance
+  - fast
+---
+
+Before the browser can request a resource from a server, it must establish a connection. Establishing a secure connection involves three steps:
+
+* Look up the domain name and resolve it to an IP address.
+
+* Set up a connection to the server.
+
+* Encrypt the connection for security.
+
+In each of these steps the browser sends a piece of data to a server and the server sends back a response. That journey, from origin to destination and back, is called a [round trip](https://developer.mozilla.org/en-US/docs/Glossary/Round_Trip_Time_(RTT)). 
+
+Depending on network conditions, a single round trip might take a significant amount of time. The connection setup process might involve up to three round trips—and more in unoptimized cases. 
+
+Taking care of all that ahead of time makes applications feel much faster. This post explains how to achieve that with two resource hints: `<rel=preconnect>` and `<rel=dns-prefetch>`.
+
+## Establish early connections with `rel=preconnect`
+
+Modern browsers [try their best to anticipate](https://www.igvita.com/posa/high-performance-networking-in-google-chrome/#tcp-pre-connect) what connections a page will need, but they cannot reliably predict them all. The good news is that you can give them a (resource 😉) hint.
+
+Adding `rel=preconnect` to a `<link>` informs the browser that your page intends to establish a connection to another domain, and that you'd like the process to start as soon as possible. Resources will load more quickly because the setup process has already been completed by the time the browser requests them. 
+
+Resource hints are named so because they are not mandatory instructions. They provide the information about what you'd like to happen, but it's ultimately up to the browser to decide whether to execute them. Setting up and keeping a connection open is a lot of work, so the browser might choose to ignore resource hints or execute them partially depending on the situation.
```

Comment:
```suggestion
Resource hints get their name because they are not mandatory instructions. They provide the information about what you'd like to happen, but it's ultimately up to the browser to decide whether to execute them. Setting up and keeping a connection open is a lot of work, so the browser might choose to ignore resource hints or execute them partially depending on the situation.
```
A little more idiomatic.